continuousflex/conda.yaml

```diff
@@ -5,17 +5,17 @@ dependencies:
   - python=3.8
   - pip:
       - setuptools==59.5.0
-      - torch==1.10.1
+      - torch==1.13.1
```

Comment:
This one should be reverted to (1.10.1).. the code doesn't work on cuda 11 recommended by scipion